src/marqo/vespa/models/feed_response.py

```diff
@@ -4,17 +4,21 @@
 
 
 class FeedDocumentResponse(BaseModel):
+    """A response from feeding a document to Vespa."""
     path_id: str = Field(alias='pathId')
     id: str
 
 
 class FeedBatchDocumentResponse(BaseModel):
+    """A response from feeding a document to Vespa while the
+    document is part of a batch."""
     status: int
-    path_id: str = Field(alias='pathId')
+    path_id: Optional[str] = Field(alias='pathId', default=None)
```

Comment:
Why is path_id nullable now? Is this change backward compatible?

Reply:
Actually, I should delete this attribute because it is not used anywhere in our code except test.

Reply:
An error response would not return a path_id so I make it optional. It shouldn't be an issue because this field is not used anywhere in the code base except tests.